frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.tsx

```diff
@@ -167,922 +78,29 @@ export const ProfileForm: React.FC<ProfilePageProps> = ({
-                  </ul>
-                </div>
-              )}
-            </div>
-          </TableOfContents.Section>
-          <TableOfContents.Section
-            id="skills-section"
-            data-h2-padding="b(bottom, xxl)"
-          >
-            <div style={{ display: "flex", alignItems: "baseline" }}>
-              <TableOfContents.Heading
-                icon={LightningBoltIcon}
-                style={{ flex: "1 1 0%" }}
-              >
-                {intl.formatMessage({
-                  defaultMessage: "My skills and experience",
-                  description: "Title of the My skills and experience section",
-                })}
-              </TableOfContents.Heading>
-              <Link
-                href={paths.skillsAndExperiences()}
-                title=""
-                {...{
-                  "data-h2-font-color": "b(lightpurple)",
-                }}
-              >
-                {intl.formatMessage({
-                  defaultMessage: "Edit Skills and Experience",
-                  description:
-                    "Text on link to update a users skills and experience",
-                })}
-              </Link>
-            </div>
-            {!experiences || experiences?.length === 0 ? (
-              <div
-                data-h2-bg-color="b(lightgray)"
-                data-h2-padding="b(all, m)"
-                data-h2-radius="b(s)"
-              >
-                <p>
-                  {intl.formatMessage({
-                    defaultMessage:
-                      "You haven't added any information here yet.",
-                    description:
-                      "Message that the user hasn't filled out the section yet",
-                  })}
-                </p>
-                <p>
-                  {intl.formatMessage(
-                    {
-                      defaultMessage:
-                        "There are <redText>required</redText> fields missing.",
-                      description:
-                        "Message that there are required fields missing. Please ignore things in <> tags.",
-                    },
-                    {
-                      redText,
-                    },
-                  )}{" "}
-                  <a href={paths.skillsAndExperiences()}>
-                    {intl.formatMessage({
-                      defaultMessage: "Click here to get started.",
-                      description:
-                        "Message to click on the words to begin something",
-                    })}
-                  </a>
-                </p>
-              </div>
-            ) : (
-              <div data-h2-padding="b(all, m)" data-h2-radius="b(s)">
-                <ExperienceSection
-                  experiences={experiences?.filter(notEmpty)}
-                />
-              </div>
-            )}
-          </TableOfContents.Section>
-        </TableOfContents.Content>
-      </TableOfContents.Wrapper>
+      <UserProfile
```

Comment:
Should salary be using localizedConstansts?
![image](https://user-images.githubusercontent.com/8978655/171457617-04fe9589-b8ec-4a8e-ac07-6a863be291a1.png)
Also, looks like language evaluation might be using an enum directly.
![image](https://user-images.githubusercontent.com/8978655/171457773-6cc62f6e-aca5-4d46-966b-6c3d03d90ea3.png)

Reply:
I'm actually not sure about this one. Are you able to point us in the right direction? We just used the existing components and im not sure if we accidentally knocked something loose while reorganizing things.

Reply:
Thanks! Resolved with [fix language and salary ranges](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2910/commits/3c2ecbd3d23c525c603510e9108f162afab47774) and [fix import path for constants](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2910/commits/e41701a7a2dcd8ae8eb5ec5e175e4175bf67ced9)